src/web/core/src/main/java/org/geoserver/web/GeoServerHomePage.java

```diff
@@ -154,7 +154,10 @@ private void homeInit() {
 
         Locale locale = getLocale();
```

Comment:
I am trying this line in homeInit() to retrieve the correct locale - however it is being called too earlier?